### PR TITLE
[FTI-3323]add include_uri_captures_in_opa_input option to OPA plugin

### PR DESCRIPTION
**Pull request by windmgc**
### Summary

Add a new option `include_uri_captures_in_opa_input` to the OPA plugin.
The new option will enable the ability to make regex capture groups included as the input to OPA

Realted PR: https://github.com/Kong/kong-ee/pull/3270
https://github.com/Kong/kong-ee/pull/3953/files

### Reason

https://konghq.atlassian.net/browse/FTI-3323

### CODE REVIEW

**Comment by lena-larionova** [comment]
@windmgc can you please rebase your branch on `release/gateway-3.0`?

File: app/_hub/kong-inc/opa/_index.md
Comment (lena-larionova) [inline app/_hub/kong-inc/opa/_index.md:87]:
```suggestion
        If set to true, the [regex capture groups](https://docs.konghq.com/gateway/latest/reference/proxy/#using-regex-in-paths) captured on the Kong Gateway Route's path field in the current request (if any) are included as input to OPA.
```

File: app/_hub/kong-inc/opa/_index.md
Comment (lena-larionova) [inline app/_hub/kong-inc/opa/_index.md:85]:
```suggestion
      default: false
      minimum_version: "3.0.x"
```

File: app/_hub/kong-inc/opa/_index.md
Comment (windmgc) [reply app/_hub/kong-inc/opa/_index.md]: done

**Review by lena-larionova — COMMENTED** [review]
Looks good, just waiting for a merge on the linked eng PR before merging this one.

**Comment by windmgc** [comment]
Hi @lena-larionova , this PR is targeting 3.1, can you review it again? Thanks!

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for submitting this!

**Pull request merged**
